New plugin - basemap switch control (#9440)
Leaflet/Leaflet: docs/_plugins/layer-switching-controls/l-basemapcontrol.md

---
name: L.basemapControl
category: layer-switching-controls
repo: https://github.com/Urban96/L.basemapControl
author: Daniel Urban
author-url: https://github.com/Urban96
demo: https://urban96.github.io/L.basemapControl/examples/example1.html
compatible-v0:
compatible-v1: true
---

> Basemap Control with Synchronized Minimap

Reviewer: Please change this describtion

Author: Done. I hope this description better captures what the plugin does.

> Leaflet plugin that adds a control for switching between multiple basemap layers. It features a synchronized minimap preview to show the next layer in the sequence.

Reviewer: @Urban96 I would suggest something like this, because for me it was at the beginning not clear that I need to click on the minimap to switch.
But if you want to keep your description, it is fine for me and I will merge it.

```suggestion
Leaflet plugin that creates a synchronized minimap preview to switch betweeen multiple basemap layers. With a click on the minimap the next basemap layer in the sequence is applied.
```

Author: Sure, it's updated! Thanks
